Constitution.txt

```diff
@@ -1,490 +1,264 @@
+4: The Vice President of these United States shall be President of the Senate, but shall have no Vote, unless they be equally divided.
 
-5: The Senate shall chuse their other Officers, and also a President pro tempore, in the Absence of the Vice President, or when he shall exercise the Office of President of the United States.
+5: The Senate shall choose their other Officers, and also a President pro tempore, in the Absence of the Vice President, or when he shall exercise the Office of President of these United States.
 
-6: The Senate shall have the sole Power to try all Impeachments. When sitting for that Purpose, they shall be on Oath or Affirmation. When the President of the United States is tried, the Chief Justice shall preside: And no Person shall be convicted without the Concurrence of two thirds of the Members present.
+6: The Senate shall have the sole Power to try all Impeachments. When sitting for that Purpose, they shall be on Oath or Affirmation. When the President of these United States is tried, the Chief Justice shall preside: And no Person shall be convicted without the Concurrence of two thirds of the Members present.
 
-7: Judgment in Cases of impeachment shall not extend further than to removal from Office, and disqualification to hold and enjoy any Office of honor, Trust or Profit under the United States: but the Party convicted shall nevertheless be liable and subject to Indictment, Trial, Judgment and Punishment, according to Law.
+7: Judgment in Cases of impeachment shall not extend further than to removal from Office, and disqualification to hold and enjoy any Office of honor, Trust or Profit under these United States: but the Party convicted shall nevertheless be liable and subject to Indictment, Trial, Judgment and Punishment, according to Law.
 
 Section 4
-1: The Times, Places and Manner of holding Elections for Senators and Representatives, shall be prescribed in each State by the Legislature thereof; but the Congress may at any time by Law make or alter such Regulations, except as to the Places of chusing Senators.
+1: The Times, Places and Manner of holding Elections for Representatives, and for deciding the Appointments of Senators, shall be prescribed in each State by the Constitution thereof, but such Elections must occur every second Year from the date on which the State joins the Union.
 
-2: The Congress shall assemble at least once in every Year, and such Meeting shall be on the first Monday in December,5 unless they shall by Law appoint a different Day.
+2: The Congress shall assemble at least once in every Year, and such Meeting shall be on the first Monday in December, unless they shall by Law appoint a different Day. Neither house shall assemble for a period of more than one Hundred days in a year, except as may be necessary in time of emergency which threatens the immediate physical safety of the Citizens of any State.
```

Comment:
The Federal government should be slow to act. By limiting the term in which they can meet, outside wartime, we limit their ability to pass legislation that might risk violations of the rights of the People or the authority of the States.